classes/01class/exercises/c01-git04/jithinjabraham/README.md

```diff
@@ -0,0 +1,21 @@
+# c01-git04
+
+## Questions (answer inline)
+
+1. Let's suppose you remotely pushed the `my_env.txt` file above. A colleague asks you to remove this information from Git. What's your colleague worried about? The colleague is worried as this file contains sensitive information that are stored in cleartext. It is not safe or good practice to store secrets on git repos.
+
+2. If you modify the file in your workspace, then commit and push it, will it be enough to erase this password information from the repository? (It's not). Why? This will not fix the issue as the file can be retrieved via the commit history.
+
+3. If you delete the file and push it, then create a new one with the rest of the information, is it enough? (It's not). Why?
+Same answer as the previous question -  there will be versions of the file stored in the commit history of the repo.
+
+4. How to fix this? How do you remove something from Git history when it is in the remote repository? This can be fixed via the git filter-repo command which was learnt in c01-git03. 
+
+
+5. Which commands would you use? Explain what the command does. Similar to the answer in the previous exercise, the commands that could be used are:
+git filter-repo --invert-paths --path file-path 
+After the above command, the changes are to pushed to the remote branches with the below command.
```

Comment:
Jithin, do you also explain what the command does here. Thanks

Reply:
Updated answer with explanation! :)